app/assets/javascripts/admin/controllers/admin-email-advanced-test.js.es6

```diff
@@ -0,0 +1,29 @@
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Ember.Controller.extend({
+  email: null,
+  text: null,
+  elided: null,
+  format: null,
+
+  actions: {
+    run() {
+      this.set("loading", true);
+
+      ajax("/admin/email/advanced-test", {
```

Comment:
Let's add a simple acceptance test so that we can have confidence that whatever UI we have for this now is working.